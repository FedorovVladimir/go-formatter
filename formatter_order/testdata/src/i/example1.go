package main

func main() { // want "incorrect declaration order"
}

// Hello func for hello user // want "incorrect declaration order"
// without params
func Hello() {

}

// Hello2 func for hello user // want "incorrect declaration order"
// without params
func Hello2() {

}
