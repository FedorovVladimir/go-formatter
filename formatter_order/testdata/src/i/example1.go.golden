package main

// Hello func for hello user // want "incorrect declaration order"
// without params
func Hello() {

}

// Hello2 func for hello user // want "incorrect declaration order"
// without params
func Hello2() {

}

func main() { // want "incorrect declaration order"
}
